app/public/index.html

```diff
@@ -11,7 +11,8 @@
         window.location.href = 'https://app.requestly.io';
```

Comment:
This file is deprecated when we shifter to vite. Please delete this file to avoid confusion in future